### ISSUE TITLE
Improving specialty characterization of a given dataset

### ISSUE THREAD

**Issue by vojtechhuser** [OHDSI/Achilles#580]
We are working on a DQ paper and we developed an improved analysis of specialties.

Existing analysis 301 is just taking into account provider table.

We created a new analysis similar to that but that is on visit level.

Some relevant notes are here
https://rpubs.com/vojtech_huser/sx

The query is the following
```
select specialty_concept_id, count(*) 
from (
      SELECT p.specialty_concept_id   
from visit_occurrence vo  join provider p  on vo.provider_id=p.provider_id 
where vo.visit_concept_id=9202  
      ) one
group by specialty_concept_id
```

Please let us know if you have change suggestions. After some discussion here, we plan to submit a PR.

**Comment by fdefalco** [comment]
pending https://github.com/OHDSI/ETL-Synthea/issues/91

**Comment by AnthonyMolinaro** [comment]
Fixed via #638
